R/calc_hem_fields.R

```diff
@@ -104,7 +104,7 @@ calc_hem_fields <- function(vri_bem, fire, hem_fields = SSGBM.VRI.BEM::hem_field
                                           default = 0)]
 
   vri_bem[, W_Shelter_1 := fifelse(SPEC_CD_1 %in% c("BA", "BB", "BL", "C", "FDI", "H", "HM", "HW", "HX", "HXM", "P", "PA", "PL", "PLC", "PLI", "S",
-                                                    "SB", "SE", "SS", "SW", "SX", "SXE", "SXL", "SXS", "SXW", "SXX") & SPEC_PCT_1 > 40 & PROJ_AGE_1 %in% c("7","8", "9") & Elev_Threshold == 1 & Slope_Limit == 1 & CROWN_MOOSE > 35 & (Static_Sb_Bog == 0 | Static_Persist_Decid == 0), 1, 0)]
+                                                    "SB", "SE", "SS", "SW", "SX", "SXE", "SXL", "SXS", "SXW", "SXX") & SPEC_PCT_1 > 40 & PROJ_AGE_1 %in% c("7","8", "9") & Elev_Threshold == 1 & Slope_Limit == 1 & CR_CLOSURE > 35 & (Static_Sb_Bog == 0 | Static_Persist_Decid == 0), 1, 0)]
```

Comment:
Change PROJ_AGE_1 %in% c("7","8", "9") to PROJ_AGE_1 > 120

Reply:
@rwaytes There is a bunch of PROJ_AGE_1 condition that are still using character. Is it ok?
https://github.com/bcgov/SSGBM-VRI-BEM/blob/f712e8ce185170052767eee1ec323259805b2bd8/R/calc_hem_fields.R#L100
https://github.com/bcgov/SSGBM-VRI-BEM/blob/f712e8ce185170052767eee1ec323259805b2bd8/R/calc_hem_fields.R#L102-L103
https://github.com/bcgov/SSGBM-VRI-BEM/blob/f712e8ce185170052767eee1ec323259805b2bd8/R/calc_hem_fields.R#L116

Reply:
I've been meaning to switch those but it hasn't been high priority. I think this happened because the field that was originally in the Python HEM script (PROJ_AGE_CL) doesn't exist in the current version of the model. PROJ_AGE_1 has similar spelling but is a different variable (PROJ_AGE_CL is age classes, PROJ_AGE_1 is (numeric) age) so it was a mistranslation. 

PROJ_AGE_1 %in% c("4", "5", "6", "7", "8", "9") should be PROJ_AGE_1 > 60

PROJ_AGE_1 %in% c("8", "9") should be PROJ_AGE_1 > 140

PROJ_AGE_1 %in% c("3","4", "5", "6", "7", "8", "9") should be PROJ_AGE_1 > 40

I can make these changes next week